lib/cryptoexchange/exchanges/zaif/services/trades.rb

```diff
@@ -0,0 +1,32 @@
+module Cryptoexchange::Exchanges
+  module Zaif
+    module Services
+      class Trades < Cryptoexchange::Services::Market
+        def fetch(market_pair)
+          output = super(ticker_url(market_pair))
+          adapt(output, market_pair)
+        end
+
+        def ticker_url(market_pair)
+          "#{Cryptoexchange::Exchanges::Zaif::Market::API_URL}/trades/#{market_pair.base.downcase}_#{market_pair.target.downcase}"
+        end
+
+        def adapt(output, market_pair)
+          output.collect do |trade|
+            tr = Cryptoexchange::Models::Trade.new
+            tr.trade_id  = trade['tid']
+            tr.base      = market_pair.base
+            tr.target    = market_pair.target
+            tr.type      = trade['trade_type'] == 'buy' ? 'bid' : 'sell'
```

Comment:
@tmlee another [PR](https://github.com/coingecko/cryptoexchange/pull/168/files) uses `buy` and `sell`, we have to decide whether we want to use `bid` or `buy`.

Reply:
Good one. To keep it consistent is to probably use bid, ask. Any way to abstract this out?

Reply:
@dominicwong617  Any thoughts on maybe solving this in a separate issue?

Reply:
https://github.com/coingecko/cryptoexchange/issues/195

Reply:
Let's not merge until we know which one to use.

Reply:
https://money.stackexchange.com/questions/1063/can-someone-explain-a-stocks-bid-vs-ask-price-relative-to-current-price

```
Whereas, the bid and ask are the best potential prices that buyers and sellers are willing to transact at: the bid for the buying side, and the ask for the selling side. But, think of the bid and ask prices you see as "tip of the iceberg" prices. That is:
```

I think we should use buy/sell

Reply:
other exchanges trades respond we've implemented use `buy/sell`

Reply:
@Swingcloud I will merge this first, issue a fix in a separate PR